mkdocs.yml

```diff
@@ -23,35 +23,19 @@ theme:
       icon: material/toggle-switch
       name: Switch to dark mode
 
-
-
   logo: images/logo.png
   favicon: images/logo.png
   features:
     - content.code.annotate
-extra:
```

Comment:
This is for [Insiders only](https://squidfunk.github.io/mkdocs-material/setup/setting-up-site-analytics/?h=feed#was-this-page-helpful)